CRM/Civicase/Form/CaseCategoryTypeCustomField.php

```diff
@@ -63,6 +63,8 @@ public function preProcess() {
       $this
     );
 
+    $this->hasCustomFieldsAssigned = count($groupTree) !== 0;
```

Comment:
@deb1990 do we need to make this a member of the class? I don't think we do, but if we do then we need to define it at the top of the class.